### PR TITLE
#feature#10 log in form style

### PR DESCRIPTION
**Pull request by hancel-avila**
# Description
What did you implemented/Why did you implemented this:
 - We stilyzed the LogIn screen with the colors previously declared. Why? The design will be uniformly stylized by the same app convention

image

### CODE REVIEW

**Review by cortezroberto — DISMISSED** [review]
Good work

File: FlightReservation/src/screens/SignUp.js [hunk 1]
```diff
@@ -112,10 +112,17 @@ const SignUp = function ({navigation}) {
                 <View style={style.lower_content_text}>
                   <Text style={{paddingTop: 20}}>
                     Already have an account?
+                    <Text
+                      style={{
+                        color: 'blue',
+                      }}
```
Comment (DemisRincon) [inline FlightReservation/src/screens/SignUp.js:118]: Avoid inline styles

File: FlightReservation/src/screens/LogIn.js [hunk 2]
```diff
@@ -44,25 +48,59 @@ const LogIn = function ({navigation}) {
...
+                  <Text style={{color: 'blue'}}>Sign up</Text>
                 </Text>
-                <TouchableOpacity onPress={()=> {navigation.navigate("SignUp")}}>
-                  <Text>Sign up</Text>
-                </TouchableOpacity>
               </View>
             </View>
-          )}
-        </Formik>
+            {/*  */}
```
Comment (DemisRincon) [inline FlightReservation/src/screens/LogIn.js:100]: delete this comment

File: FlightReservation/src/screens/LogIn.js [hunk 3]
```diff
@@ -44,25 +48,59 @@ const LogIn = function ({navigation}) {
...
-                  Don't have an account?
+            </View>
+
+            <View>
+              <View
+                style={{
+                  flex: 1,
+                  justifyContent: 'flex-end',
+                  paddingBottom: 40,
+                }}
```
Comment (DemisRincon) [inline FlightReservation/src/screens/LogIn.js:59]: avoid inline styles

File: FlightReservation/src/screens/LogIn.js [hunk 4]
```diff
@@ -44,25 +48,59 @@ const LogIn = function ({navigation}) {
...
+                    Log In
+                  </Text>
+                </View>
+              </Pressable>
+
+              <Text
+                style={[
+                  style.lower_content_text,
+                  {paddingTop: 10, paddingBottom: 10},
+                ]}>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/LogIn.js:76]: avoid inline styles

File: FlightReservation/src/screens/LogIn.js [hunk 5]
```diff
@@ -44,25 +48,59 @@ const LogIn = function ({navigation}) {
...
+                  justifyContent: 'flex-end',
+                  paddingBottom: 40,
+                }}
+              />
+              <Pressable
+                disabled={!formProps.isValid || loading}
+                onPress={formProps.handleSubmit}
+                title="Log in">
+                <View style={style.btn}>
+                  <Text style={{color: colors.white, fontSize: 18}}>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/LogIn.js:66]: avoid inline styles

File: FlightReservation/src/screens/LogIn.js [hunk 6]
```diff
@@ -44,25 +48,59 @@ const LogIn = function ({navigation}) {
...
+                </View>
+              </Pressable>
+              <View style={style.lower_content_text}>
+                <Text
+                  style={{paddingTop: 20}}
+                  onPress={() => {
+                    navigation.navigate('SignUp');
+                  }}>
+                  Already have an account?{' '}
+                  <Text style={{color: 'blue'}}>Sign up</Text>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/LogIn.js:96]: avoid inline styles

File: FlightReservation/src/screens/SignUp.js [hunk 7]
```diff
@@ -112,10 +112,17 @@ const SignUp = function ({navigation}) {
                 <View style={style.lower_content_text}>
                   <Text style={{paddingTop: 20}}>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/SignUp.js:113]: avoid inline styles

File: FlightReservation/src/screens/SignUp.js [hunk 8]
```diff
@@ -112,10 +112,17 @@ const SignUp = function ({navigation}) {
...
                     Already have an account?
+                    <Text
+                      style={{
+                        color: 'blue',
+                      }}
+                      onPress={() => {
+                        navigation.navigate('LogIn');
+                      }}>
+                      {' '}
```
Comment (DemisRincon) [inline FlightReservation/src/screens/SignUp.js:122]: No need for this line

**Review by DemisRincon — CHANGES_REQUESTED** [review]
Please read the recommendations for improvement most of them are inline styles please move that to an stylesheet document

**Review by YilKanda — COMMENTED** [review]
As an added comment, I can recommend using internationalization instead of static text.

You can use this library, for example, to do this:

[react-native-i18n](https://www.npmjs.com/package/react-native-i18n)

**Review by yovanha — DISMISSED** [review]
Good job!

**Review by DemisRincon — DISMISSED** [review]

**Review by jkzasori — APPROVED** [review]

**Review by cortezroberto — APPROVED** [review]
LGTM!

**Pull request merged**